anvil/src/udev.rs

```diff
@@ -893,17 +916,17 @@ fn render_surface<'a>(
         {
             surface.fps_element.update_fps(surface.fps.avg().round() as u32);
             surface.fps.tick();
+            elements.push(CustomRenderElements::Fps(surface.fps_element.clone()));
```

Comment:
Not sure if this is intentionally, but putting the fps element inside the cursor check we only get the fps overlay on a output where we currently have a cursor.

Reply:
fixed